config/routes.rb

```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
   devise_for :users
-  root to: 'pages#home'
+  root to: 'pets#home'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  resources :pets do
+    resources :bookings, only: [:create]
+  end
+  get "dashbord", to: 'dashboards#display'
```

Comment:
```suggestion
  get "dashboard", to: 'dashboards#display'
```

Reply:
good catch! I will change it!

Reply:
Just commit the suggestion in one branch only, or it will conflict.